File: fib-limit.php
<?php
$a = 0;
$b = 1;
$i = 0;
$limit = 10;
while (true) {
    usleep(1000);
    $a += $b;
    echo $a, "\n";
    $i += 1;
    $b += $a;
    echo $b, "\n";
    $i += 1;
    if ($i == $limit) {
        break;
    }
}
?>

File: fib-limit.c
#include <stdio.h>

int a = 0, b = 1, i = 0;
int limit = 2;

int main() {
    while (1) {
        a += b;
        printf(a, "\n");
        i += 1;
        b += a;
        printf(b, "\n");
        i += 1;
        if (i == limit) {
            break;
        }
    }
}